src/didactWebView.ts

```diff
@@ -266,26 +266,37 @@ export class DidactWebviewPanel {
 		// And the uri we use to load this script in the webview
 		const scriptUri = scriptPathOnDisk.with({ scheme: 'vscode-resource' });
 
-		let cssFile = 'webviewslim.css';
-		if (this.isAsciiDoc) {
-			cssFile = 'asciidoctor.css';
-		}
+		const bulmaCssHtml = `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">`;
```

Comment:
Maybe we can put these lines specific to Markdown inside the else block L.287 ? (if i understood well)
I think it will be easier to understand which part is related to AsciiDoc and which is related to markdown.

Reply:
will see what I can do